src/Adapters/AwsS3Connector.php

```diff
@@ -4,6 +4,7 @@
  * This file is part of Laravel Flysystem.
  *
  * (c) Graham Campbell <graham@cachethq.io>
+ * with contributions by Raul Ruiz <publiux@gmail.com>
```

Comment:
Please don't change the copyright header. Add your name to the class level authors only.